packages/backend-app-api/src/services/implementations/httpRouter/httpRouterServiceFactory.test.ts

```diff
@@ -14,39 +14,29 @@
  * limitations under the License.
  */
 
+import {
+  ServiceFactoryTester,
+  mockServices,
+} from '@backstage/backend-test-utils';
 import { httpRouterServiceFactory } from './httpRouterServiceFactory';
 
 describe('httpRouterFactory', () => {
   it('should register plugin paths', async () => {
-    const rootHttpRouter = { use: jest.fn() };
-    const factory = httpRouterServiceFactory() as any;
+    const rootHttpRouter = mockServices.rootHttpRouter.mock();
+    const tester = ServiceFactoryTester.from(httpRouterServiceFactory(), {
```

Comment:
```suggestion
    const tester = ServiceFactoryTester.from(httpRouterServiceFactory, {
```

just because we can! :D